sdk/go/auto/minimum_version.go

```diff
@@ -19,6 +19,6 @@ import "github.com/blang/semver"
 var minimumVersion = calculateMinVersion()
 
 func calculateMinVersion() semver.Version {
-	v, _ := semver.ParseTolerant("3.1.0")
+	v, _ := semver.ParseTolerant("3.1.0-alpha")
```

Comment:
I believe this should be 3.2.0-alpha

Reply:
It's kind of painful to bump the minimum required CLI version - that will break some existing users - are we sure we want to do that here - vs. handling older CLIs and just doing "better" on newer CLIs?

Reply:
Honestly, even after everything I said about why it's fine to bump min versions, I really do struggle with justifying the minimum version bump for such a small specific change. It would take some very minor refactoring to be able to do this check per feature rather than an overall version bump. 

I think it's fine to defer to if/until we hear more feedback but can we please open an issue to track making that improvement and hold ourselves to doing that work if bumping this version becomes a regular occurrence?

Reply:
I'm fine with designing a new policy, but I would ask that we do it outside of the context of a code review if we really feel this to be an important topic.

I've opened an issue here: https://github.com/pulumi/pulumi/issues/6937

Let's continue the discussion there.